static/app/components/events/eventEntries.tsx

```diff
@@ -74,10 +75,17 @@ function EventEntries({
   }
 
   const hasContext = !objectIsEmpty(event.user ?? {}) || !objectIsEmpty(event.contexts);
+  const hasActionableItems = actionableItemsEnabled({
+    eventId: event.id,
+    organization,
+    projectSlug,
+  });
 
   return (
     <div className={className}>
-      <EventErrors event={event} project={project} isShare={isShare} />
```

Comment:
Can you delete this component now? Don't think it's used anywhere else

Reply:
yup, will delete it in a separate pr